tests/conftest.py

```diff
@@ -24,3 +26,61 @@ def fixture_basic_outlier_dataframe():
     Creates basic example data for use in tests
     """
     return pd.DataFrame({"example_feature": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]})
+
+
+@pytest.fixture(name="basic_outlier_csv")
+def fixture_basic_outlier_csv(
+    tmp_path: pathlib.Path, basic_outlier_dataframe: pd.DataFrame
+):
+    """
+    Creates basic example data csv for use in tests
+    """
+
+    basic_outlier_dataframe.to_csv(csv_path := tmp_path / "example.csv", index=False)
```

Comment:
```suggestion
    basic_outlier_dataframe.to_csv(csv_path := tmp_path / "basic_example.csv", index=False)
```

Reply:
but maybe this suggestion is getting a bit ahead of ourselves... (if we intend to have other examples beyond basic...) prob ok to do in a future PR when/if we do add more example data tests

Reply:
I think this is a good idea, thanks, I'll add it in.